functional-samples/cookbook.sidepanel-action/README.md

```diff
@@ -0,0 +1,15 @@
+# Site-specific side panel example
```

Comment:
Should this be a in a sidepanel-site-specific folder since that's the main thing it's showing? The action just seems like a side thing?

Reply:
Good idea 👍